src/wazuh_db/helpers/wdb_global_helpers.c

```diff
@@ -1061,6 +1062,80 @@ int wdb_remove_group_from_belongs_db(const char *name, int *sock) {
     return result;
 }
 
+int wdb_set_agent_groups_csv(int id, char* groups_csv, char* mode, char* sync_status, char* source, int *sock) {
+    cJSON* j_groups = cJSON_CreateObject();
```

Comment:
This JSON object is only checked but never used to save data. We should remove it.

Reply:
You are right. Fixed.